web/modules/custom/collection_page/collection_page.behat.inc

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/**
+ * @file
+ * Contains \CollectionPageSubContext.
+ */
+
+use Behat\Gherkin\Node\TableNode;
+use Drupal\DrupalExtension\Context\DrupalSubContextBase;
+use Drupal\DrupalExtension\Context\DrupalSubContextInterface;
+
+/**
+ * Behat step definitions for testing collections.
+ */
+class CollectionPageSubContext extends DrupalSubContextBase implements DrupalSubContextInterface {
+  /**
+   * Test collections.
+   *
+   * @var \Drupal\node\Entity\Node[]
+   */
+  protected $collectionPages = [];
+
+  /**
+   * The bundle type of the content entity.
+   *
+   * @var String
+   */
+  protected $bundle = 'collection_page';
+
+  /**
+   * Creates content of a given type provided in the form.
+   * @codingStandardsIgnoreStart
+   * | title    | author     | status | created           |
+   * | My title | Joe Editor | 1      | 2014-10-17 8:00am |
+   * | ...      | ...        | ...    | ...               |
+   * @codingStandardsIgnoreEnd
```

Comment:
This doesn't seem to violate coding standards, those indicators are probably not needed.